source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -303,15 +303,23 @@ def wrapper(*args, **kwargs):
             if objects[1] != 200 or 'journeys' not in objects[0]:
                 return objects
             for journey in objects[0]['journeys']:
+                args = dict(request.args)
+                ids = {o['stop_point']['stop_area']['id']
+                       for s in journey.get('sections', []) if 'from' in s
+                       for o in (s['from'], s['to']) if 'stop_point' in o}
+                if 'region' in kwargs:
+                    args['region'] = kwargs['region']
                 if "sections" not in journey:#this mean it's an isochrone...
-                    args = dict(request.args)
                     if 'to' not in args:
                         args['to'] = journey['to']['id']
                     if 'from' not in args:
                         args['from'] = journey['from']['id']
-                    if 'region' in kwargs:
-                        args['region'] = kwargs['region']
                     journey['links'] = [create_external_link('v1.journeys', rel='journeys', **args)]
+                elif ids and 'public_transport' in (s['type'] for s in journey['sections']):
+                    args['min_nb_journeys'] = 5
+                    ids.update(args.get('allowed_id[]', []))
+                    args['allowed_id[]'] = list(ids)
+                    journey['links'] = [create_external_link('v1.journeys', rel='other_schedules', **args)]
```

Comment:
I would loved `same_journey_shoot_again`

but i know you are not so funny. So you can use `same_journey_schedules`